.tekton/determine-image-tag-task-oci-ta.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: determine-image-tag-oci-ta
+  namespace: rh-acs-tenant
+# TODO(ROX-23812): Refactor to a task bundle
+spec:
+  description: Determines the tag for the output image using the StackRox convention from 'make tag' output.
+  params:
+  - name: tag-suffix
+    description: Suffix to append to generated image tag.
+    type: string
+  - name: SOURCE_ARTIFACT
+    description: The Trusted Artifact URI pointing to the artifact with
+      the application source code.
+    type: string
+  results:
+  - name: image-tag
+    description: Image Tag determined by custom logic.
```

Comment:
[nit] The style of naming params is inconsistent. In this case, `SOURCE_ARTIFACT` looks odd against the others. Should it be `source-artifact` or should rather other params go `SCREAMING_CASE` to match the style of Konflux tasks?

I'm actually also ok if you keep things as they are now because params style is mixed in Konflux tasks themselves but at least `SOURCE_ARTIFACT` is universally like that in all.

Reply:
Let's do `SCREAMING_CASE`!

Reply:
Done in https://github.com/stackrox/stackrox/pull/11616/commits/21469bc3ca7b0a706d1adbebf5ac316117177184.

I did not touch the `determine-image-tag` task (without the OCI artifacts), as it will be removed in a future PR.